spec/fixtures/iocage_list-t

```diff
@@ -0,0 +1,2 @@
+JID   UUID                                  BOOT  STATE  TAG
+-     b439bd7a-5376-11e7-ad91-d979c2a3eb55  off   down   f11-puppet4
```

Comment:
Is this output taken from the new iocage version?  I must have missed the `-t` option.